client-resttemplate/publishing.gradle

```diff
@@ -0,0 +1,28 @@
+def garProjectId = System.getenv("GOOGLE_CLOUD_PROJECT")
```

Comment:
I decided to just make a new file since `artifactory.gradle` didn't seem to fit anymore